dessia_common/workflow/core.py

```diff
@@ -1941,6 +1943,119 @@ def _method_jsonschemas(self):
         warnings.warn("method_jsonschema method is deprecated. Use method_schema instead", DeprecationWarning)
         return self.method_schemas
 
+    @staticmethod
+    def generate_object_input_string(objects):
+        """ Generate input string for object(s). """
+        signature = "('Set your arguments here')"
+        if len(objects) > 1:
+            instances = [f"\n\t{o.__class__.__name__}{signature}" for o in objects]
+            return f"[{','.join(instances)}\n]"
+        return f"{objects[0].__class__.__name__}{signature}"
+
+    @staticmethod
+    def generate_file_input_string(type_files):
+        """ Generate input string for file(s). """
+        signature = ".from_file('Set your filepath here')"
+        if len(type_files) > 1:
+            instances = [f"\n\t{o.__class__.__name__}{signature}" for o in type_files]
+            return f"[{','.join(instances)}\n]"
+        return f"{type_files[0].__class__.__name__}{signature}"
+
+    @staticmethod
+    def is_object_sequence(sequence):
+        """ Checks if a sequence contains serializable objects. """
+        return any(isinstance(element, SerializableObject) for element in sequence)
+
+    def process_sequence(self, sequence):
```

Comment:
This could also be a function, if the others are set as functions.

Consider setting them in workflow/utils.py

Reply:
done !